encryption.rst

```diff
@@ -113,6 +121,74 @@ like so.
    $ sudo -E {command} build --encrypt encrypted.sif encrypted.def
    Starting build...
 
+Image info verification (``Privileged Encryption``)
+---------------------------------------------------
+
+.. code:: 
+
+   $ {command} sif list encrypted.sif
+
+Example:
+
+.. code:: 
+
+   ------------------------------------------------------------------------------
+   ID   |GROUP   |LINK    |SIF POSITION (start-end)  |TYPE
+   ------------------------------------------------------------------------------
+   1    |1       |NONE    |32176-32223               |Def.FILE
+   2    |1       |NONE    |32223-34202               |JSON.Generic
+   3    |1       |NONE    |34202-34292               |JSON.Generic
+   4    |1       |NONE    |36864-17608704            |FS (Encrypted squashfs/*System/arm64)
+   5    |1       |4       |17608704-17609449         |Cryptographic Message (PEM/RSA-OAEP)
+
+Note that ``ID=4`` partition has the type of ``Encrypted squashfs``
```

Comment:
I am sorry I didn't notice this earlier, but I have one more little change in four places.  For this use
```
Note that partition 4 is type ``Encrypted squashfs``.
```

Reply:
no problem at all, thanks for the comments
https://github.com/apptainer/apptainer-userdocs/pull/179/commits/816c972aee9ca359516da01997481793ca62c2ca